src/rgw/store/dbstore/sqlite/sqliteDB.cc

```diff
@@ -331,9 +331,12 @@ static int get_objectdata(const DoutPrefixProvider *dpp, DBOpInfo &op, sqlite3_s
   blob = sqlite3_column_blob(stmt, 3);
   datalen = sqlite3_column_bytes(stmt, 3);
 
-  char data[datalen+1] = {};
-  if (blob)
+  char data[datalen];
```

Comment:
Shouldn't this be "char data[datalen+1] "

Reply:
Fixed, thanx for noticing.